cinder/tests/unit/volume/drivers/vmware/test_vmware_vmdk.py

```diff
@@ -3350,12 +3366,31 @@ def test_migrate_volume(self, get_moref, vops, backing=None,
             else:
                 self.assertEqual((True, None), ret_val)
 
+            if capabilities and 'location_info' in capabilities:
+                if vmdk.LOCATION_DRIVER_NAME in capabilities['location_info']:
```

Comment:
Why `in` and not `startswith()` or even better checking `capabilities['location_info'].split(':')`? I don't like `in` because it might be at any place. Let it be a user we use. I know it's improbable, but still.